4-appfactory/apps/cymbal-bank/main.tf

```diff
@@ -29,8 +30,10 @@ locals {
   expanded_app_services = flatten([
     for key, services in local.app_services : [
       for service in services : {
-        app_name     = key
-        service_name = service
+        app_name            = key
+        app_acronym         = "cb"
```

Comment:
Just for the record: If we want to remove this hardcoded value, it could be retrieved from the `apps` variable or the 2-multitenant remote state